eng/Build.props

```diff
@@ -24,7 +24,7 @@
       Format:
       <DependencyPackageProjects Include="$(RepoRoot)src\referencePackages\src\**\Microsoft.Extensions.Options.5.0.0.csproj" />
     -->
-    <DependencyPackageProjects Include="$(RepoRoot)src\textOnlyPackages\src\microsoft.private.intellisense\8.0.0-preview-20230828.1\Microsoft.Private.Intellisense.8.0.0-preview-20230828.1.csproj" />
+    <DependencyPackageProjects Include="$(RepoRoot)src\textOnlyPackages\src\microsoft.private.intellisense\8.0.0-preview-20230918.1\Microsoft.Private.Intellisense.8.0.0-preview-20230918.1.csproj" />
```

Comment:
@MichaelSimons - For a version upgrade like this, is this acceptable to change the version in Build.props like this?